app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -215,8 +215,8 @@ class SectionRenderingService(frontendAppConfig: FrontendAppConfig, lookupRegist
       pageLevelErrorHtml,
       frontendAppConfig,
       actionForm,
-      retrievals.renderSaveAndComeBackLater,
-      retrievals.continueLabelKey,
+      retrievals.renderSaveAndComeBackLater && !formTemplate.draftRetrievalMethod.isNotPermitted,
```

Comment:
this is hard to read, can add a method/variable thats describes what its supposed to mean?